flytekit/bin/entrypoint.py

```diff
@@ -280,6 +280,13 @@ def setup_execution(
             )
         cb = cb.with_serialization_settings(ssb.build())
 
+    debugger_on = get_one_of("FLYTE_INTERNAL_START_DEBUGPY")
```

Comment:
can we make this Env var little smaller, we have been going towards the following convention
`__FF_DBGPY`

Reply:
Added `__FF_DBGPY` and `__FF_DBGPY_PORT`